source/api_documentation/network_integration/promo_numbers/index.rst

```diff
@@ -110,6 +110,20 @@ When you POST a promo number, we return the promo_number in the response body. T
     - array of strings
     - an array of stringified limiters on the boundaries of where to look for local numbers given as npa or npa nxx pairs (ex. [“805”, “805612”])
 
+  * - promo_type
+    - string (optional)
+    - For international campaigns that support local or mobile numbers. If not passed then promo numbers will default to the organizational default. (US/CA campaigns will continue to use local_center for local numbers)
+
+      "Local"
+
+      "Mobile"
```

Comment:
@jebentier is resolving this in an in progress ticket, so we are good leaving the documentation as is